modules/mcuboot/CMakeLists.txt

```diff
@@ -109,6 +108,12 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
         )
     endif()
 
+    if(DEFINED SIGN_ARG_IMAGE_DEPENDENCIES)
+      set(sign_dependencies "-d;\"${SIGN_ARG_IMAGE_DEPENDENCIES}\"")
+    else()
+      set(sign_dependencies "")
+    endif()
```

Comment:
You don't need this `else` clause right? An unset variable in cmake is just empty.

Reply:
Removed